### ISSUE TITLE
Flutter BKASH IOS

### ISSUE THREAD

**Issue by BlessFI** [codeboxrcodehub/flutter-bkash#15]
I recently implemented the sdk on Android, which is working perfectly, however, the SDK does not seem to be working on IOS. The Webview opens, but nothing renders on the Page. When building the App on XCode, there is no Pods for flutter_bkash.

I have attached a screenshot 
![PXL_20241108_224044080](https://github.com/user-attachments/assets/78ade3a4-67e4-412f-aa24-65fa761ea0ef)

**Comment by manchumahara** [comment]
I believe we never tested on ios, the package was tested in android and never finished for ios, isn't it @rdnasim ?

**Comment by rdnasim** [comment]
Right, We tested android device. It's working fine. But not tested iOS yet.

**Comment by rdnasim** [comment]
Testing on iOS devices has been completed. The application is functioning properly on both Android and iOS platforms.
